src/System.Private.Xml/tests/XmlResolver/System.Xml.XmlResolver.Tests/XmlPreloadedResolverConstructionTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Xml.Resolvers;
+using Xunit;
+
+namespace System.Xml.XmlResolver.Tests
+{
+    public class XmlPreloadedResolverConstructionTests
+    {
+        [Fact]
+        public void XmlResolverWithDefaultConstructor()
+        {
+            var resolver = new XmlPreloadedResolver();
+            Assert.Equal(14, resolver.PreloadedUris.Count());
```

Comment:
I do not have strong push on testing the exact values but since we are doing it everywhere else... (unless it covered somewhere further)